templates/parachain/README.md

```diff
@@ -63,14 +63,9 @@ You still need a relaychain node - you can download the `polkadot`
 (and the accompanying `polkadot-prepare-worker` and `polkadot-execute-worker`)
 binaries from [Polkadot SDK releases](https://github.com/paritytech/polkadot-sdk/releases/latest).
 
-Make sure to bring the parachain node - as well as `polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`,
-and `zombienet` - into `PATH` like so:
-
-```sh
-export PATH="./target/release/:$PATH"
-```
-
-This way, we can conveniently use them in the following steps.
+Make sure to bring `zombienet` and the relaychain binaries
+(`polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`)
```

Comment:
```suggestion
(`polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`, `parachain-template-node`)
```

Reply:
Maybe keeping the `export PATH` example is a good idea.

Reply:
really though? seems super inconvenient?

compared to just having the path in the zombienet config itself?

Reply:
better then imo to run `cargo install` so that the binary is placed into the `cargo bin`

Reply:
There is one more argument supporting the PATH path.

The user could be trying out the readme, having downloaded the `parachain-template-node` binary from [a release](https://github.com/paritytech/polkadot-sdk-parachain-template/releases) instead of building it.

Then, the hardcoded path in zombienet wouldn't make sense, but instructions to have it in PATH would still hold.

Reply:
> better then imo to run `cargo install` so that the binary is placed into the `cargo bin`

OK, I've added a step with `cargo install` for the parachain binary.
And, I left an example with `export PATH` for the other binaries.